src/main/java/org/opentripplanner/routing/api/request/refactor/preference/VehicleRentalPreferences.java

```diff
@@ -0,0 +1,119 @@
+package org.opentripplanner.routing.api.request.refactor.preference;
+
+import java.io.Serializable;
+import org.opentripplanner.routing.api.request.RoutingRequest;
+import org.opentripplanner.routing.vehicle_rental.VehicleRentalStation;
+
+public class VehicleRentalPreferences implements Cloneable, Serializable {
+
+  private boolean allow = false;
```

Comment:
This should probably be in VehicleRentalRequest

Reply:
I combined CarPreferences.parkAndRide & BikePreferences.parkAndRide into one and moved them into RentalRequest.

Reply:
Moved allow as well.